public/style.css

```diff
@@ -14,4 +16,55 @@
 
 ul {
   list-style: none;
+  overflow: hidden;
+  justify-content: center;
+}
+
+li {
+  display: inline-block;
+  width: 30%;
+  margin-top: 5%;
+  margin-right: 5px;
+  text-align: left;
+  padding: 1% 0% 1% 5%;
+  background-color: grey;
+  color: white;
+  border-radius: 25px;
+  font-family: 'Kdam Thmor Pro', sans-serif;
+  font-size: large;
```

Comment:
Maybe you can add `cursor: pointer` to the li, so that it is clear you can click on the `li`